src/test/fixtures/profiles/example-profile.js

```diff
@@ -165,6 +165,7 @@ const contentProcessProfile = {
   threads: [
     Object.assign({ name: 'GeckoMain', processType: 'tab' }, thread),
   ],
+  processes: [],
```

Comment:
without this change, the test suite would then fail with the changes below -- which is expected :)